### ISSUE TITLE
Create "open" FEC page in Wagtail

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3316]
### Summary

**What we are after:**
By federal law, the FEC must have an "open government" page specific to the FEC. We currently have two pages that combine to meet this need, but one is on classic. We need to combine them into one page that covers all required information in our current design. The first iteration of the page will be just a re-creation of the material.

### Current pages
Classic: https://classic.fec.gov/open/ 
Transition: https://transition.fec.gov/open/index.shtml

### Completion criteria

_List action items that need to be done to complete this task. List in checklist formatting._
- [x] Draft text for page; review by @AmyKort 
- [x] Create page under "About the FEC"
~- [ ] Update link in footer for "Open Government" (Content may be able to do this with coaching and review by front end.)~ - We'll do this as part of #3345 

### Future work

- #3345 We need to add content to display all the things we've been doing with fec.gov

**Comment by johnnyporkchops** [comment]
@JonellaCulmer , Do you think we could consider having some kind of modern looking OPENFec logo or representative visual image that could go adjacent to text on this page. We have alot of text heavy pages and a visual on this page would give it more "flare" (not in the Office Space sense of the word :-)).

**Comment by dorothyyeager** [comment]
@johnnyporkchops @JonellaCulmer That kind of thing might be good for a future iteration. My understanding from @AmyKort is that the agency needs to get this page up quickly to comply with federal requirements, so for our first go-round, the goal is to keep this task simple.

**Comment by dorothyyeager** [comment]
I sent the draft text for the new page to @AmyKort for review.

**Comment by dorothyyeager** [comment]
I've got the draft of the first iteration of the page ready for review by @AmyKort and @PaulClark2 .

**Comment by dorothyyeager** [comment]
I've made @PaulClark2's edits. I need to ask @AmyKort about the reports we're linking in one section, and then we should be good to publish this soon.

**Comment by dorothyyeager** [comment]
Talked with Amy. We will run this page by the Staff Director's office to ensure we've got all reports that should be in the MVP version, and then publish.

**Comment by dorothyyeager** [comment]
We've decided to go ahead and publish the first iteration of the page, which basically brings over the content from the classic site, with a couple of items updated so they didn't give incorrect information.

The new page is at https://www.fec.gov/about/open-government-federal-election-commission/.

Future work to update the page more completely and link it under "Open" on the footer will take place in #3345. Closing this ticket as the MVP work is complete.
